manifests/key.pp

```diff
@@ -15,7 +15,7 @@
     command     => "/usr/sbin/dnssec-keygen -a HMAC-MD5 -r /dev/urandom -b 128 -n USER ${name}",
     cwd         => "${cfg_dir}/bind.keys.d",
     require     => [
-      Package['dnssec-tools','bind9'],
+      Package['dnssec-tools',$package],
```

Comment:
So.. shouldn't it be dns::server::package or something?

Alternatively I would argue that this should just require 'dnssec-tools' and let the other classes require their own packages?